docs/articles/documentation/using-testcafe/programming-interface/TestCafe.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: docs
+title: TestCafe Class
+permalink: /documentation/using-testcafe/programming-interface/TestCafe/
+---
+# TestCafe Class
+
+A TestCafe server instance.
+
+To create it, use the [createTestCafe](/testcafe/documentation/using-testcafe/programming-interface/global/#createTestCafe) global function.
+
+```js
+var createTestCafe = require('testcafe');
+var testCafe = await createTestCafe('localhost', 1337, 1338);
+```
```

Comment:
Use our formatting for code snippets (align eq-sign on the longest declaration). Use ES6 var declaration keywords in code snippets (`const` and `let`). The rule is the following: for any immutable variable use `const`, in all other cases use `let`. We don't do that in our source code currently, but it's a subject for the further refactoring